roles/osp_observability/templates/prometheus.yml.j2

```diff
@@ -1,9 +1,7 @@
-#jinja2: trim_blocks:False
 global:
   scrape_interval: {{ prometheus_scrape_interval }}
   external_labels:
     agent_host: {{ ansible_fqdn }}
-    {% endif -%}
```

Comment:
Yeah so apparently I screwed this up during a merge conflict resolution, but there was no CI to catch it at the time. :)